src/openshift/cluster.ts

```diff
@@ -172,7 +173,7 @@ export class Cluster extends OpenShiftItem {
             .then(() => Cluster.odo.execute(Command.odoLoginWithUsernamePassword(clusterURL, username, passwd)))
             .then((result) => Cluster.save(username, passwd, password, result))
             .then((result) => Cluster.loginMessage(clusterURL, result))
-            .catch((error) => Promise.reject(`Failed to login to cluster '${clusterURL}' with '${error}'!`));
+            .catch((error) => Promise.reject(`Failed to login to cluster '${clusterURL}' with '${Filters.filterPassword(error.cmd)}'!`));
```

Comment:
by doing this we are loosing error message and printing out only command line with filtered password.